qiita_pet/templates/study_base.html

```diff
@@ -15,12 +16,26 @@
 
   function fill_artifact(aid) {
     $("#study-main").html("ARTIFACT " + aid + " HTML STUFF");
+    $.get("/admin/artifact/", {artifact_id: +aid })
```

Comment:
We are performing 2 checks, once at python level on other at JS level - is it needed? This goes back to the question above from @antgonza. My opinion is that if you are already making sure that at JS level you're passing an integer, you don't need to perform the check in the python code.